webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -36,7 +36,7 @@ var userOverride = {
             'https://googleupload.applozic.com/files/url?key={key}', // generate viewable link for a file incase of file upload on google cloud
         notificationIconLink: '',
         notificationSoundLink: '',
-        mapStaticAPIkey: 'AIzaSyCWRScTDtbt8tlXDr6hiceCsU83aS2UuZw',
+        mapStaticAPIkey: 'AIzaSyCTGgLQbsvMotNNjRqPnWCln4y4LcBvxxE',
```

Comment:
can you please check other repos too? and see if we need to update those as well.

Reply:
checked in km-dashboard, km-docs, km-website, km-server